digit-health-services/health-sync-service/src/test/java/org/digit/health/sync/service/FileSyncServiceTest.java

```diff
@@ -0,0 +1,139 @@
+package org.digit.health.sync.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.digit.health.sync.helper.SyncUpRequestTestBuilder;
+import org.digit.health.sync.kafka.Producer;
+import org.digit.health.sync.service.checksum.MD5Checksum;
+import org.digit.health.sync.service.compressor.GzipCompressor;
+import org.digit.health.sync.web.models.SyncLog;
+import org.digit.health.sync.web.models.request.SyncUpDto;
+import org.digit.health.sync.web.models.request.SyncUpMapper;
+import org.digit.health.sync.web.models.request.SyncUpRequest;
+import org.egov.tracer.model.CustomException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.io.*;
+import java.nio.file.Files;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+
+@ExtendWith(MockitoExtension.class)
+class FileSyncServiceTest {
+
+    @Mock
+    private Producer producer;
+
+    @Mock
+    private FileStoreService fileStoreService;
+
+    @Mock
+    private ObjectMapper objectMapper;
+
+    @Mock
+    private GzipCompressor compressor;
+
+    @Mock
+    private MD5Checksum checksumValidator;
+
+
+    @InjectMocks
+    private FileSyncService fileSyncService;
+
+    @BeforeEach
+    void setUp() {
+        objectMapper = new ObjectMapper();
+        fileSyncService = new FileSyncService(producer,fileStoreService,objectMapper,compressor,checksumValidator);
+    }
+
+    @Test
+    @DisplayName("ShouldSuccessfullyCallChecksumValidatorAndFileStoreService")
```

Comment:
Change it to sentence with all lower case

Reply:
Done